web_development_101/project_git_basics.md

```diff
@@ -66,7 +66,7 @@ We used this command to commit the *README.md* file `git commit -m "Add README.m
   <a href="http://imgur.com/a/2B4Lw"><img class="tutorial-img" src="http://i.imgur.com/YvYwHXM.png" title="source: imgur.com" /></a>
 3. Back in your terminal enter `git status`, notice that *README.md* is now un-staged again.
   <a href="https://imgur.com/AlUO59b"><img class="tutorial-img" src="https://i.imgur.com/AlUO59b.png" title="source: imgur.com" /></a>
-4. Add *README.md* to the staging area with `git add README.md`
+4. Add *README.md* to the staging area with `git add README.md`.
```

Comment:
Although grammatically correct, we should leave off the period after the `git add` command so that student's aren't confused if they should include it or not